src/hooks/chat.js

```diff
@@ -78,6 +79,11 @@ export default function() {
       })
     })
 
+    // Post an Item Quality/Flaw
+    html.on("click", ".item-property", ev => {
+      WFRP_Utility.postProperty(ev.target.text)
+    })
+
```

Comment:
Is there a reason this was added back? I'm not accepting JQuery as I'm phasing it out with Foundry, but it doesn't seem to be needed in the first place.